sway/config/bar.c

```diff
@@ -39,7 +49,11 @@ void free_bar_config(struct bar_config *bar) {
 	free(bar->status_command);
 	free(bar->font);
 	free(bar->separator_symbol);
-	// TODO: Free mouse bindings
+	while (bar->bindings->length) {
+		struct bar_binding *binding = bar->bindings->items[0];
+		list_del(bar->bindings, 0);
```

Comment:
This seems unnecessary, just iterate over the list and free each item without deleting.